virttest/vt_resmgr/resources/storage/nfs/nfs_volume.py

```
@@ -0,0 +1,66 @@
+        """
+        for node in nodes:
+            _, backing_id = self.get_binding(node)
+            if not backing_id:
+                LOG.debug(
+                    f"The nfs volume has already unbound from the backing on {node.name}"
+                )
+                continue
+
+            r, o = node.proxy.resource.destroy_resource_backing(backing_id)
+            if r != 0:
+                raise Exception(o["out"])
```

Ruff (0.14.10)

Create your own exception

(TRY002)